src/components/GradeBookContentContainer.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import AddButton from './MainActionButton';
+
+const ContentContainer = styled.div`
+	color: #002366;
+	font-weight: bold;
+`;
+
+const TitleRow = styled.div`
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	width: 100%;
+	border-style: hidden hidden solid hidden;
+	color: #828282;
+`;
+
+const ColumnItemHeader = styled.div`
+	display: flex;
+	flex-direction: column;
+	flex-basis: 100%;
+	flex: 1;
+	color: #002366;
+`;
+
+const ButtonColumn = styled.div`
+	display: flex;
+	flex-direction: column;
+	flex-basis: initial;
+	margin: 1.5em;
+	width: 15em;
+`;
+
+const AddItemButton = styled(AddButton)`
+	margin: 1.5em;
+	color: #002366;
+`;
+
+const CourseItemRow = styled.div`
+	justify-content: center;
+	display: flex;
+	flex-direction: row;
+	max-width: 100%;
+	white-space: nowrap;
+`;
+
+const Table = styled.table`
+	width: 100%;
+	color: #828282;
+	text-align: left;
+	text-decoration: underline;
+	font-weight: normal;
+	font-size: small;
+`;
+
+const CourseItemTitleSpace = styled.th`
+	padding: 2em;
+`;
+
+const CourseItemTitle = styled.th`
+	padding: 4em;
+`;
+
```

Comment:
I think we should reduce these values. The spacing container I believe should be `0.25em` and the others should just have a top padding of `1em`. I think the left aligned text is more like what we want, and the spacing does not need to be that much as there will be spacing in the container that holds this component as well.

Reply:
Hi Braiden, just changed the numbers, is this what you mean? :)